src/site/content/en/blog/speed-at-scale/index.md

```diff
@@ -0,0 +1,85 @@
+
+Web pages often contain a large number of images, which contribute to data-usage, [page-bloat](https://httparchive.org/reports/state-of-images) and how fast a page can load. Many of these images are offscreen, requiring a user to scroll in order to view them. 
+
+Until now, you've needed to solve lazy-loading images using a JavaScript library but that may soon change. This summer, Chrome will be [launching](https://www.youtube.com/embed/YJGCZCaIZkQ?start=705) support for the [loading](https://addyosmani.com/blog/lazy-loading/) attribute which brings native `<img>` and `<iframe>` lazy-loading to the web!.
+
+<figure class="w-figure">
+  <img src="./Speed-at1.png" alt="Native lazy-loading highlighting offscreen content being loaded on-demand" loading="lazy">
+</figure>
+
+The `loading` attribute allows a browser to defer loading offscreen images and iframes until users scroll near them. `loading` supports three values:
+
+* `lazy`: is a good candidate for lazy loading.
+* `eager`: is not a good candidate for lazy loading. Load right away.
+* `auto`: browser will determine whether or not to lazily load.
+
+```html
+<img src="io2019.jpg" loading="lazy" alt="..." />
+<iframe src="video-player.html" loading="lazy"></iframe>
+```
+
+The exact heuristics for "when the user scrolls near" is left up to the browser. In general, our hope is that browsers will start fetching deferred images and iframe content a little before it comes into the viewport. 
+
+The `loading` attribute is implemented behind flags in Chrome Canary. You can try out [this demo](https://mathiasbynens.be/demo/img-loading-lazy) in Chrome 75+ with the `chrome://flags/#enable-lazy-image-loading` and `chrome://flags/#enable-lazy-frame-loading` flags turned on. 
+
+A [write-up](https://addyosmani.com/blog/lazy-loading/) on the native lazy-loading feature is available with more details.
+
+## Google fonts now supports font-display as a query parameter
+
+We announced that we would finally be bringing support for [font-display](https://font-display.glitch.me) to [Google Fonts](https://fonts.google.com). We are happy to share this is now available in production for all Google Fonts users via the new [display](https://developers.google.com/fonts/docs/getting_started#use_font-display) query-string parameter.
+
+```html
+https://fonts.googleapis.com/css?family=Lobster&display=swap
+``` 
+
+The `font-display` descriptor lets you decide how your web fonts will render or fallback, depending on how long it takes for them to load. It supports a number of values including `auto`, `block`, `swap`, `fallback` and `optional`.
+
+Previously, the only way to specify font-display for web fonts from Google Fonts was to self-host them but this change removes the need to do so. 
+
+The Google Fonts documentation has been updated to include `font-display` in the default code embeds. We hope this will encourage more developers to try out this exciting addition.
+
+<figure class="w-figure">
```

Comment:
What is this image showing? It's not entirely clear to me. Maybe it could be better introduced?

Reply:
The image is showing the default embed code window from Google Fonts. I've tried to update the text to note that the embed we are referring to is in the image below it. Does that help?